subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitor.java

```diff
@@ -56,6 +57,7 @@ public class JavaProcessStackTracesMonitor {
     private static final Pattern WINDOWS_PID_PATTERN = Pattern.compile("([0-9]+)\\s*$");
     private static final Pattern UNIX_PID_PATTERN = Pattern.compile("([0-9]+)");
 
+    @Nullable
```

Comment:
This class needs to be run in command line directly with Java 11 scripting, like this:

```
/opt/jdk/open-jdk-11/bin/java subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/timeout/JavaProcessStackTracesMonitor.java .
```

Is `javax.annotation.Nullalble` part of JDK distribution, without any 3rd party packages?

Reply:
Also not intentional changes, will revert. I don't know how I ended up pushing all this crap... I think I messed up an interactive revert or something...